src/modules/customers/customers.controller.ts

```diff
@@ -34,6 +35,22 @@ export class CustomersController {
     return this.customersService.getCustomers(backendQuery);
   }
 
+  @Get('dun-and-bradstreet')
```

Comment:
```suggestion
  @Get('dun-bradstreet')
```

Please, this is to keep it aligned with multiword endpoints i.e. `interest-rate`, `exposure-period` and etc.